api/app/users/api.py

```diff
@@ -92,6 +95,10 @@ def post(self):
         except IntegrityError:
             return EMAIL_IN_USE
 
+        send_mail(recipient=user.email,
+                  subject='Password verification for IndabaDeepLearning portal',
```

Comment:
Let's change the subject to `Email verification for Deep Learning Indaba Portal`

Can we also use an environment variable/config for the URL?